### PR TITLE
adding CORS and SameSite cookie requirement to user manual

### PR DESCRIPTION
**Pull request by x-du**
I am adding some instructions for CORS and SameSite cookie requirements into the user manual.

### CODE REVIEW

**Review by elinmstr — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -4,6 +4,66 @@
 
 ## Getting Started
 
+### Requirements for Embedding
+
+If you plan to use your own dossier in the Playground, please meet the following prerequisites.
+
+#### Enable Cross-Origin Resource Sharing (CORS)
+
+Cross-Origin Resource Sharing (CORS) provides a way for a web application running in one origin (domain, protocol, and port) to access selected resources from a server in a different origin. A cross-origin HTTP request uses additional HTTP headers to tell the browser to let the web application share resources. For security reasons, browsers restrict cross-origin HTTP requests initiated from within scripts. This means that when a web application requests HTTP resources from a different origin, the response from the other origin must include the right CORS headers.
```
Comment (magua-io) [inline README.md:13]: from or within?

```suggestion
Cross-Origin Resource Sharing (CORS) provides a way for a web application running in one origin (domain, protocol, and port) to access selected resources from a server in a different origin. A cross-origin HTTP request uses additional HTTP headers to tell the browser to let the web application share resources. For security reasons, browsers restrict cross-origin HTTP requests initiated from scripts. This means that when a web application requests HTTP resources from a different origin, the response from the other origin must include the right CORS headers.
```

File: README.md [hunk 2]
```diff
@@ -4,6 +4,66 @@
...
+
+To enable CORS for the Playground to embed:
+
+1. Open the Library Admin page. Your URL should be similar to the following:
+
+    ```
+    https://<host_name>:<port>/MicroStrategyLibrary/admin
+    ```
+
```
Comment (magua-io) [inline README.md:22]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -4,6 +4,66 @@
...
+2. Navigate to Library Server -> Security Settings.
+3. Allow Library to be embedded in the Playground using url `https://microstrategy.github.io`. You could check `All`, but whitelisting is recommended. 
+    ![picture 1](images/CORS_config.png)  
+4. Restart your Library Server.
+
+
+
```
Comment (magua-io) [inline README.md:29]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -4,6 +4,66 @@
...
+Using the Library Admin page is the easiest way to enable CORS for the REST Server, but you can also configure CORS manually.
+
+1. Navigate to `MicroStrategyLibrary/WEB-INF/classes/config/configOverride.properties`.
+2. Edit the configOverride.properties file in a text editor.
+3. Add the following lines, or replace them if already present:
+    ```
```
Comment (magua-io) [inline README.md:35]:
```suggestion

    ```
```

File: README.md [hunk 5]
```diff
@@ -18,6 +78,7 @@ You can also use your dossier by entering your dossier URL after selecting _Your
 
 ![Use Your Dossier](./images/use-your-dossier.gif)
 
+
```
Comment (magua-io) [inline README.md:81]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -4,6 +4,66 @@
...
+4. Restart your MicroStrategy Library Server.
+
+#### SameSite Cookie
+
+Google Chrome (version 80+) and Microsoft Edge (version 86+) introduced new changes that may impact embedding.
+
+For Embedding SDK to function as expected in a 3rd party context, it is required to explicitly label session cookies with `SameSite=None; Secure`. Please perform the following steps on your MicroStrategy Library Server machine. 
+
+1. If context.xml doesn't already exist in the following folder location, create it:
+MicroStrategyLibrary\META-INF\context.xml
```
Comment (magua-io) [inline README.md:49]:
```suggestion
1. If `context.xml` doesn't already exist in the following folder location, create it at `MicroStrategyLibrary\META-INF\context.xml`
```

**Review by magua-io — APPROVED** [review]
LGTM. Please review the comments and apply changes accordingly. After that, you can merge.

**Pull request merged**